backend/tests/conftest.py

```diff
@@ -1499,6 +1502,18 @@ def mock_graphql(
     gql_factory.build.return_value = mock_client
 
 
+@pytest.fixture(scope="function")
+def mock_failing_gql(
+    mocker,
+):
+    gql_factory.set_url({"SUBGRAPH_ENDPOINT": "http://localhost:12345"})
```

Comment:
I've replaced this URL with "domain.example" so that intent is clear. This URL is not resolved, but `gql` checks if some URL is provided on init time.